docs/files.md

```diff
@@ -746,3 +747,26 @@ A file's watermark can be removed by calling
 ```js
 client.files.removeWatermark('67890', callback);
 ```
+
+Get Representation
+------------------
+
+A file's representation can be retrieved by calling
+[`files.getRepresentations(fileID, [representationTypes], options,
+callback)`](https://opensource.box.com/box-node-sdk/Files.html#getRepresentation).
+You will be able to fetch a pdf representation, thumbnail representation, multi-page images
+representation, and fetching text representation.
+
+You can fetch different dimensions for the representation by calling
+```js
+client.files.getRepresentationInfo('67890', client.files.representation.PDF, null, callback);
+```
+Additionally there are two header options you can include. set_content_disposition_type and set_content_disposition_filename. set_content_disposition_type must be set to either inline
```

Comment:
What do these options mean for Node.js?  It would be good to explain how the behavior differs for each option, since I think these are mostly for use in a browser context.